packages/examples/packages/json-rpc/snap.manifest.json

```diff
@@ -26,5 +26,6 @@
       }
     }
   },
+  "platformVersion": "6.9.0",
```

Comment:
```suggestion
```

Reply:
This is inserted by the CLI tests 🤔 Need to see why it's still doing that.